### PR TITLE
consolidated the two integration test contract origination scripts

### LINKED ISSUE #1937: combine the two contract origination scripts in the examples folder
There are two scripts in the examples folder that both originate contracts needed for tests.
There only needs to be one script.

Include the integration tests script into the live code in the docs example script.

### PR DESCRIPTION
**Pull request by michaelkernaghan**
Close #1937 

there are two integration test contract origination scripts but its better to have them just be one.

### CODE REVIEW

File: example/deploy-originated-integration-tests-contracts.ts [hunk 1]
```diff
@@ -132,7 +136,12 @@ async function checkBalances(users: string | any[]) {
 async function originateTheContracts() {
   await checkBalances(users);
 
-  console.log('originating...');
+  console.log('originating...')
+  contract_catalogue.set('KnownContract', await originateKnownContract());
+  contract_catalogue.set('knownBigMapContract', await originateknownBigMapContract());
+  contract_catalogue.set('SaplingStateContract', await originateSaplingStateContract());
+  contract_catalogue.set('Tzip12BigMapOffChain', await originateTzip12BigMapOffChain());
```
Comment (roxaneletourneau) [inline example/deploy-originated-integration-tests-contracts.ts:143]: Are these contracts used for the examples on the website?

Comment (roxaneletourneau) [reply example/deploy-originated-integration-tests-contracts.ts:143]: I thought this file was use to deploy the contracts used on the website. Why is it renamed to example/deploy-originated-integration-tests-contracts.ts?

Comment (michaelkernaghan) [reply example/deploy-originated-integration-tests-contracts.ts:143]: I have changed the name again to:  deploy-contracts-for-website-and-integration-tests.ts

Comment (michaelkernaghan) [reply example/deploy-originated-integration-tests-contracts.ts:143]: i will add indications that separate thecontracts for the integration tests from the contracts for the website.

Comment (michaelkernaghan) [reply example/deploy-originated-integration-tests-contracts.ts:143]:
```
 //Contracts used in integration tests. Use these in ~/taquito/integration-tests/config.ts
  console.log('Originating contracts used in integration tests...')
  contract_catalogue.set('KnownContract', await originateKnownContract());
  contract_catalogue.set('knownBigMapContract', await originateknownBigMapContract());
  contract_catalogue.set('SaplingStateContract', await originateSaplingStateContract());
  contract_catalogue.set('Tzip12BigMapOffChain', await originateTzip12BigMapOffChain());

  //Contracts used in live code examples in the Taquito Docs Website
  console.log('Originating contracts used in the Taquito docs Website...')
  contract_catalogue.set('IncrementContract', await originateIncrementContract());
  contract_catalogue.set('MichelsonMapContract', await originateMichelsonMap());
  contract_catalogue.set('LambdaViewContract', await originateLambda1());
```

**Comment by roxaneletourneau** [comment]
Closing because we have the script "originate-known-contracts.ts" in the integration test folder to deploy the contracts used in the integration tests. We don't need a duplicated script in the examples folder. 
example/originate-known-contracts.ts has already been deleted in another PR

**Pull request closed without merging**